src/Integration/UserSettingsConfiguration/UserSettingsProvider.cs

```diff
@@ -61,8 +65,8 @@ public UserSettingsProvider(
                     return;
                 }
 
-                globalSettingsStorage.SettingsFileChanged += OnSettingsFileFileChanged;
-                solutionSettingsStorage.SettingsFileChanged += OnSettingsFileFileChanged;
+                globalSettingsStorage.SettingsFileChanged += OnSettingsFileChanged;
+                solutionSettingsStorage.SettingsFileChanged += OnSettingsFileChanged;
                 // The subscription to the ActiveSolutionTracker events should happen after the ISolutionSettingsStorage is initialized,
```

Comment:
Something I thought of just now and is not directly related to your PR: https://blog.stephencleary.com/2009/06/threadsafe-events.html

We probably should consider moving away from built in events in favor of IObservable pattern for better thread safety and other customization